pkg/cmd/create.go

```diff
@@ -164,9 +186,9 @@ func createAction(cmd *cobra.Command, args []string) error {
 	}
 	clusterStackPath := args[0]
 
-	if mode != stableMode && mode != hashMode {
+	if mode != stableMode && mode != hashMode && mode != customMode {
 		fmt.Println("The mode is ", mode)
-		return fmt.Errorf("mode is not supported please choose from - stable, hash")
+		return fmt.Errorf("mode is not supported please choose from - stable, hash, custom mode")
 	}
```

Comment:
Sorry, you missunderstood me.

I meant: Remove that: fmt.Println("The mode is ", mode)

use: return fmt.Errorf("mode %q is not supported please choose from - stable, hash or custom", mode)